Marlin/src/libs/numtostr.cpp

```diff
@@ -413,7 +413,7 @@ const char* ftostr72rj(const_float_t f) {
 const char* ftostr52sp(const_float_t f) {
   long i = INTFLOAT(f, 2);
   uint8_t dig;
-  conv[1] = MINUSOR(i, ' ');
+  conv[1] = RJDIGIT(i, 100000);
```

Comment:
Whatever is printing in a non-ideal way, maybe it should use `ftostr52sprj` or another num-to-string function that better fits.